packages/playground/remote/service-worker.ts

```diff
@@ -98,11 +98,19 @@ initializeServiceWorker({
 			// @see controlledIframe below for more details.
 			if (
 				// WordPress Core version of block-editor.js
-				unscopedUrl.pathname.endsWith('/js/dist/block-editor.js') ||
-				unscopedUrl.pathname.endsWith('/js/dist/block-editor.min.js') ||
+				unscopedUrl.pathname.endsWith(
+					'/wp-includes/js/dist/block-editor.js'
+				) ||
+				unscopedUrl.pathname.endsWith(
+					'/wp-includes/js/dist/block-editor.min.js'
+				) ||
 				// Gutenberg version of block-editor.js
-				unscopedUrl.pathname.endsWith('build/block-editor/index.js') ||
-				unscopedUrl.pathname.endsWith('build/block-editor/index.min.js')
+				unscopedUrl.pathname.endsWith(
+					'/gutenberg/build/block-editor/index.js'
```

Comment:
Gutenberg may be installed in any directory, not just `gutenberg`. Is this change necessary?

Reply:
No, the first one was what caused the Yoast issue. I just added the `gutenberg` check to avoid similar issues in the future. 

I can remove that part.